src/room/useLoadGroupCall.ts

```diff
@@ -112,12 +117,17 @@ export const useLoadGroupCall = (
             createPtt ? "PTT" : "video"
           } call`
         );
-        return await client.createGroupCall(
-          room.roomId,
-          createPtt ? GroupCallType.Voice : GroupCallType.Video,
-          createPtt,
-          GroupCallIntent.Room
-        );
+        return await client
+          .createGroupCall(
+            room.roomId,
+            createPtt ? GroupCallType.Voice : GroupCallType.Video,
+            createPtt,
+            GroupCallIntent.Room
+          )
+          .then((groupCall) => {
+            groupCall.statsCollectIntervalTime = STATS_COLLECT_INTERVAL_TIME_MS;
+            return groupCall;
+          });
```

Comment:
I would probably just do `const newGroupCall = await client.createGroupCall(...); newGroupCall. statsCollectIntervalTime = STATS_COLLECT_INTERVAL_TIME_MS; return newGroupCall;` at this point, as I found my self trying to work out whether the `await` was still necessary above.